core/src/main/bin/questdb.sh

```diff
@@ -57,7 +57,11 @@ function read_link {
 }
 
 function usage {
-    echo "Usage: $0 start|status|stop [-f] [-d path] [-t tag]"
+    if [ "$(uname)" == "Darwin" ]; then
+        echo "Usage: $0 start|status|stop [-f] [-d path] [-t tag] [-mof max-open-files]"
```

Comment:
would prefer if the command line options and behaviour would be the same on all OS.
why not to use always the system level limits as we do for all other operating systems?

Reply:
Sounds good. I think it would be better if it can always use the system level limits. However, I don't have another OS for testing, as I use macos only. Do you mind to try to make it compatible with other operating systems?

Reply:
Oh, do you mean that I should remove the -mof option? Just always use the system level limit

Reply:
yes, that is what i meant.
let me test the change and approve.
thank you.